projects/addin-client/src/src/addin-client.service.ts

```diff
@@ -70,10 +79,24 @@ export class AddinClientService {
    */
   public settingsClick: EventEmitter<any> = new EventEmitter(true);
 
-  constructor() {
+  constructor(
+    rendererFactory: RendererFactory2,
+    themeService: SkyThemeService
+  ) {
     this.addinClient = new AddinClient({
       callbacks: {
         init: (args: AddinClientInitArgs) => {
+          if (args.themeSettings) {
```

Comment:
Do we need to `init` the `themeService` regardless of whether the theme settings are passed in--so it can respond to theme changes later? Just wondering what happens if `args.themeSettings` is undefined here then the `themeChange` callback is invoked later. Or is that a nonsense case?

Reply:
I believe in practice it'll be a non-issue since I wouldn't expect the theme to change if one is not sent at initialization.  I haven't been able to come up with a case where the theme isn't set at init, but good to guard against just in case.  I could sanity check in the `theme-change` case if you like, but probably not necessary.